tests/25_solute_dg_bc/01_inflow.yaml

```diff
@@ -32,7 +32,7 @@ problem: !Coupling_Sequential
           bc_conc: 3
         - region: domain
           diff_m: !FieldFormula
-            value: ' ((0) if (y>0.5) else (1e-2)) '
+            value: '[ [((0) if (y>0.5) else (1e-2)), 0, 0], [0, ((0) if (y>0.5) else (1e-2)), 0], [0, 0, ((0) if (y>0.5) else (1e-2))] ]'
```

Comment:
Rewrite as "scalar * eye(3)", use manual unit tensor until `eye` is supported by bparser.